lib/reversetunnel/srv.go

```diff
@@ -636,6 +544,55 @@ func (s *server) Shutdown(ctx context.Context) error {
 	return trace.Wrap(err)
 }
 
+// HandleRequest processes global out-of-band requests. Global out-of-band
+// requests are processed in order (this way the originator knows which
+// request we are responding to). If Teleport does not support the request
+// type or an error occurs while processing that request Teleport will reply
+// req.Reply(false, nil).
+//
+// For more details: https://tools.ietf.org/html/rfc4254.html#page-4
+func (s *server) HandleRequest(ctx context.Context, r *ssh.Request) {
+	switch r.Type {
+	case teleport.NetconfigRequest:
+		s.handleNetConfigRequest(ctx, r)
+	default:
+		if r.WantReply {
```

Comment:
`Reply` to a request that doesn't want a reply is a noop internally to x/crypto/ssh, so you can avoid this check and always `r.Reply(false, nil)`.